src/store/index.ts

```diff
@@ -21,6 +21,9 @@ export function createStore( services: QueryBuilderServices ): Store<RootState>
 				valueData: {
 					value: '',
 				},
+				propertyValueRelationData: {
+					value: '',
```

Comment:
same here when we are using the enum:

`propertyValueRelation: {
					value: PropertyValueRelation.Matching,
				},`

Reply:
updated